tests/verify_package_files.py

```diff
@@ -36,12 +36,26 @@
 DEFAULT_PACKAGE_DIR = HERE / "test_data" / "package-files"
 
 PYPI_PATH = "https://files.pythonhosted.org/packages"
-WHEEL_PATH = f"{PYPI_PATH}/py3/a/ansible/ansible-{{version}}-py3-none-any.whl"
-SDIST_PATH = f"{PYPI_PATH}/source/a/ansible/ansible-{{version}}.tar.gz"
 
 ANTSIBULL_BUILD = os.environ.get("ANTSIBULL_BUILD", "antsibull-build")
 PLACEHOLDER_ANTSIBULL_VERSION = "(ANTSIBULL_VERSION)"
 
+NO_COLOR = bool(os.environ.get("NO_COLOR"))
+FORCE_COLOR = bool(os.environ.get("FORCE_COLOR"))
+BOLD = "\033[1m"  # ]
+RED = "\033[31m"  # ]
+GREEN = "\033[32m"  # ]
+CLEAR = "\033[0m"  # ]
```

Comment:
What are these comments for?

Reply:
They're there so (n)vim's indent plugin doesn't get all messed up.

Reply:
Somehow reminds me of spacebar heating ;)